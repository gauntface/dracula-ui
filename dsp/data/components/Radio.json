{
  "dsp_spec_version": "0.0.1",
  "last_updated_by": "System",
  "last_updated": "2020-10-07T00:01:16.431Z",
  "entities": [
    {
      "class": "component",
      "type": "page",
      "id": "Radio",
      "name": "Radio",
      "last_updated": "2020-10-07T00:01:16.431Z",
      "last_updated_by": "System",
      "description": "Radio is a styled HTML Input of type radio.\n\nThere are no behavior changes applied to the native HTML tag other\nthan light styling done via CSS in order to keep Radios accessible.",
      "related_entity_ids": [],
      "tags": ["component"],
      "snippets": {
        "trigger": "Radio",
        "languages": {
          "javascript": "<div>\n  <div style={{...}}>\n    <Radio id=\"one\" defaultChecked={true} color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"one\" className=\"drac-text drac-text-white\">\n      Option 1\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"two\" color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"two\" className=\"drac-text drac-text-white\">\n      Option 2\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"three\" color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"three\" className=\"drac-text drac-text-white\">\n      Option 3\n    </label>\n  </div>\n</div>",
          "html": "<div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"one\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n      checked=\"\"\n    /><label for=\"one\" class=\"drac-text drac-text-white\">Option 1</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"two\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n    /><label for=\"two\" class=\"drac-text drac-text-white\">Option 2</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"three\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n    /><label for=\"three\" class=\"drac-text drac-text-white\">Option 3</label>\n  </div>\n</div>\n"
        }
      },
      "ext_com_draculaui_variations": [
        {
          "title": "Usage",
          "html": "<div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"one\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n      checked=\"\"\n    /><label for=\"one\" class=\"drac-text drac-text-white\">Option 1</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"two\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n    /><label for=\"two\" class=\"drac-text drac-text-white\">Option 2</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"three\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n    /><label for=\"three\" class=\"drac-text drac-text-white\">Option 3</label>\n  </div>\n</div>\n",
          "react": "<div>\n  <div style={{...}}>\n    <Radio id=\"one\" defaultChecked={true} color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"one\" className=\"drac-text drac-text-white\">\n      Option 1\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"two\" color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"two\" className=\"drac-text drac-text-white\">\n      Option 2\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"three\" color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"three\" className=\"drac-text drac-text-white\">\n      Option 3\n    </label>\n  </div>\n</div>",
          "docs": "",
          "screenshot": "./dsp/assets/Radio.png"
        },
        {
          "title": "Disabled",
          "html": "<div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"one\"\n      color=\"white\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-white\"\n      checked=\"\"\n    /><label for=\"one\" class=\"drac-text drac-text-white\">Checked</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"two\"\n      color=\"white\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-white\"\n    /><label for=\"two\" class=\"drac-text drac-text-white\">Unchecked</label>\n  </div>\n</div>\n",
          "react": "<div>\n  <div style={{...}}>\n    <Radio id=\"one\" defaultChecked={true} color=\"white\" name=\"demo\" />\n    <label htmlFor=\"one\" className=\"drac-text drac-text-white\">\n      Checked\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"two\" color=\"white\" name=\"demo\" />\n    <label htmlFor=\"two\" className=\"drac-text drac-text-white\">\n      Unchecked\n    </label>\n  </div>\n</div>",
          "docs": "",
          "screenshot": "./dsp/assets/RadioDisabled.png"
        },
        {
          "title": "Colors",
          "html": "<div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"black\"\n      color=\"black\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-black\"\n    /><label for=\"black\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"white\"\n      color=\"white\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-white\"\n    /><label for=\"white\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"cyan\"\n      color=\"cyan\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-cyan\"\n    /><label for=\"cyan\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"green\"\n      color=\"green\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-green\"\n    /><label for=\"green\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"orange\"\n      color=\"orange\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-orange\"\n    /><label for=\"orange\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"pink\"\n      color=\"pink\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-pink\"\n    /><label for=\"pink\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"purple\"\n      color=\"purple\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-purple\"\n    /><label for=\"purple\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"red\"\n      color=\"red\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-red\"\n    /><label for=\"red\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"yellow\"\n      color=\"yellow\"\n      name=\"demo\"\n      class=\"drac-radio drac-radio-yellow\"\n    /><label for=\"yellow\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"animated\"\n      color=\"animated\"\n      name=\"demo\"\n      class=\"drac-radio\"\n    /><label for=\"animated\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"cyanGreen\"\n      color=\"cyanGreen\"\n      name=\"demo\"\n      class=\"drac-radio\"\n    /><label for=\"cyanGreen\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"pinkPurple\"\n      color=\"pinkPurple\"\n      name=\"demo\"\n      class=\"drac-radio\"\n    /><label for=\"pinkPurple\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"purpleCyan\"\n      color=\"purpleCyan\"\n      name=\"demo\"\n      class=\"drac-radio\"\n    /><label for=\"purpleCyan\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n  <div style=\"margin-bottom: 10px\">\n    <input\n      type=\"radio\"\n      id=\"yellowPink\"\n      color=\"yellowPink\"\n      name=\"demo\"\n      class=\"drac-radio\"\n    /><label for=\"yellowPink\" class=\"drac-text drac-text-white\">Radio</label>\n  </div>\n</div>\n",
          "react": "<div>\n  <div style={{...}}>\n    <Radio id=\"black\" color=\"black\" name=\"demo\" />\n    <label htmlFor=\"black\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"white\" color=\"white\" name=\"demo\" />\n    <label htmlFor=\"white\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"cyan\" color=\"cyan\" name=\"demo\" />\n    <label htmlFor=\"cyan\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"green\" color=\"green\" name=\"demo\" />\n    <label htmlFor=\"green\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"orange\" color=\"orange\" name=\"demo\" />\n    <label htmlFor=\"orange\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"pink\" color=\"pink\" name=\"demo\" />\n    <label htmlFor=\"pink\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"purple\" color=\"purple\" name=\"demo\" />\n    <label htmlFor=\"purple\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"red\" color=\"red\" name=\"demo\" />\n    <label htmlFor=\"red\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"yellow\" color=\"yellow\" name=\"demo\" />\n    <label htmlFor=\"yellow\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"animated\" color=\"animated\" name=\"demo\" />\n    <label htmlFor=\"animated\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"cyanGreen\" color=\"cyanGreen\" name=\"demo\" />\n    <label htmlFor=\"cyanGreen\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"pinkPurple\" color=\"pinkPurple\" name=\"demo\" />\n    <label htmlFor=\"pinkPurple\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"purpleCyan\" color=\"purpleCyan\" name=\"demo\" />\n    <label htmlFor=\"purpleCyan\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n  <div style={{...}}>\n    <Radio id=\"yellowPink\" color=\"yellowPink\" name=\"demo\" />\n    <label htmlFor=\"yellowPink\" className=\"drac-text drac-text-white\">\n      Radio\n    </label>\n  </div>\n</div>",
          "docs": "",
          "screenshot": "./dsp/assets/RadioColors.png"
        }
      ],
      "ext_com_draculaui_props": {
        "color": {
          "defaultValue": null,
          "description": "The Dracula UI theme color for the radio button.",
          "name": "color",
          "parent": {
            "fileName": "src/components/Radio/Radio.tsx",
            "name": "RadioProps"
          },
          "required": true,
          "type": {
            "name": "\"white\" | \"cyan\" | \"green\" | \"orange\" | \"pink\" | \"purple\" | \"red\" | \"yellow\" | \"purpleCyan\" | \"yellowPink\" | \"cyanGreen\" | \"pinkPurple\" | \"animated\""
          }
        },
        "name": {
          "defaultValue": null,
          "description": "The name of the radio. Mirrors the name HTML attribute.",
          "name": "name",
          "parent": {
            "fileName": "src/components/Radio/Radio.tsx",
            "name": "RadioProps"
          },
          "required": false,
          "type": { "name": "string | undefined" }
        }
      },
      "ext_com_draculaui_docgen": {
        "description": "Radio is a styled HTML Input of type radio.\n\nThere are no behavior changes applied to the native HTML tag other\nthan light styling done via CSS in order to keep Radios accessible.",
        "displayName": "Radio",
        "methods": [],
        "props": {
          "color": {
            "defaultValue": null,
            "description": "The Dracula UI theme color for the radio button.",
            "name": "color",
            "parent": {
              "fileName": "src/components/Radio/Radio.tsx",
              "name": "RadioProps"
            },
            "required": true,
            "type": {
              "name": "\"white\" | \"cyan\" | \"green\" | \"orange\" | \"pink\" | \"purple\" | \"red\" | \"yellow\" | \"purpleCyan\" | \"yellowPink\" | \"cyanGreen\" | \"pinkPurple\" | \"animated\""
            }
          },
          "name": {
            "defaultValue": null,
            "description": "The name of the radio. Mirrors the name HTML attribute.",
            "name": "name",
            "parent": {
              "fileName": "src/components/Radio/Radio.tsx",
              "name": "RadioProps"
            },
            "required": false,
            "type": { "name": "string | undefined" }
          }
        }
      }
    }
  ]
}
