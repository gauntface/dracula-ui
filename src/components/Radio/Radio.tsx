import { ColorMap } from '@/base/colors'
import React, { HTMLAttributes } from 'react'

export const radioColors: ColorMap = {
  black: 'drac-radio-black',
  white: 'drac-radio-white',
  cyan: 'drac-radio-cyan',
  green: 'drac-radio-green',
  orange: 'drac-radio-orange',
  pink: 'drac-radio-pink',
  purple: 'drac-radio-purple',
  red: 'drac-radio-red',
  yellow: 'drac-radio-yellow',
  animated: '',
  cyanGreen: '',
  pinkPurple: '',
  purpleCyan: '',
  yellowPink: ''
}

export interface RadioProps extends HTMLAttributes<HTMLInputElement> {
  color: keyof typeof radioColors
  name?: string
}

export const Radio: React.FC<RadioProps> = (props) => {
  const finalProps = {
    ...props,
    className: `drac-radio ${radioColors[props.color]}`
  }

  return <input type="radio" {...finalProps} />
}

Radio.displayName = 'Radio'
