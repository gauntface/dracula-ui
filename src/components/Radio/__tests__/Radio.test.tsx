import React from 'react'
import * as ReactDOM from 'react-dom'
import { Radio } from '@/components/Radio/Radio'

describe('Radio', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Radio color="white" />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})
