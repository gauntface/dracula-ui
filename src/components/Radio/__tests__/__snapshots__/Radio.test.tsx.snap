// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site: Radio Basic Usage 1`] = `
Object {
  "docs": "Radio buttons let a user select only one of a limited number of choices.",
  "html": "<div>
  <input
    type=\\"radio\\"
    id=\\"one\\"
    color=\\"purple\\"
    name=\\"demo\\"
    class=\\"drac-radio drac-radio-purple\\"
    checked=\\"\\"
  /><label for=\\"one\\" class=\\"drac-text drac-text-white\\">Checked</label>
</div>
",
  "react": "<div>
  <Radio id=\\"one\\" defaultChecked={true} color=\\"purple\\" name=\\"demo\\" />
  <label htmlFor=\\"one\\" className=\\"drac-text drac-text-white\\">
    Checked
  </label>
</div>;
",
  "title": "Usage",
}
`;

exports[`Site: Radio Colors 1`] = `
Object {
  "docs": "Radio can be customized to use any of the Dracula UI theme colors.",
  "html": "<div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"white\\"
      color=\\"white\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-white\\"
    /><label for=\\"white\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"cyan\\"
      color=\\"cyan\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-cyan\\"
    /><label for=\\"cyan\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"green\\"
      color=\\"green\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-green\\"
    /><label for=\\"green\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"orange\\"
      color=\\"orange\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-orange\\"
    /><label for=\\"orange\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"pink\\"
      color=\\"pink\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-pink\\"
    /><label for=\\"pink\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"purple\\"
      color=\\"purple\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-purple\\"
    /><label for=\\"purple\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"red\\"
      color=\\"red\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-red\\"
    /><label for=\\"red\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"yellow\\"
      color=\\"yellow\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-yellow\\"
    /><label for=\\"yellow\\" class=\\"drac-text drac-text-white\\">Radio</label>
  </div>
</div>
",
  "react": "<div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"white\\" color=\\"white\\" name=\\"demo\\" />
    <label htmlFor=\\"white\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"cyan\\" color=\\"cyan\\" name=\\"demo\\" />
    <label htmlFor=\\"cyan\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"green\\" color=\\"green\\" name=\\"demo\\" />
    <label htmlFor=\\"green\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"orange\\" color=\\"orange\\" name=\\"demo\\" />
    <label htmlFor=\\"orange\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"pink\\" color=\\"pink\\" name=\\"demo\\" />
    <label htmlFor=\\"pink\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"purple\\" color=\\"purple\\" name=\\"demo\\" />
    <label htmlFor=\\"purple\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"red\\" color=\\"red\\" name=\\"demo\\" />
    <label htmlFor=\\"red\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"yellow\\" color=\\"yellow\\" name=\\"demo\\" />
    <label htmlFor=\\"yellow\\" className=\\"drac-text drac-text-white\\">
      Radio
    </label>
  </div>
</div>;
",
  "title": "Colors",
}
`;

exports[`Site: Radio Disabled 1`] = `
Object {
  "docs": "A radio select is unusable and un-clickable.",
  "html": "<div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"one\\"
      color=\\"white\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-white\\"
      checked=\\"\\"
    /><label for=\\"one\\" class=\\"drac-text drac-text-white\\">Checked</label>
  </div>
  <div style=\\"margin-bottom: 10px\\">
    <input
      type=\\"radio\\"
      id=\\"two\\"
      color=\\"white\\"
      name=\\"demo\\"
      class=\\"drac-radio drac-radio-white\\"
    /><label for=\\"two\\" class=\\"drac-text drac-text-white\\">Unchecked</label>
  </div>
</div>
",
  "react": "<div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"one\\" defaultChecked={true} color=\\"white\\" name=\\"demo\\" />
    <label htmlFor=\\"one\\" className=\\"drac-text drac-text-white\\">
      Checked
    </label>
  </div>
  <div style={{ marginBottom: 10 }}>
    <Radio id=\\"two\\" color=\\"white\\" name=\\"demo\\" />
    <label htmlFor=\\"two\\" className=\\"drac-text drac-text-white\\">
      Unchecked
    </label>
  </div>
</div>;
",
  "title": "Disabled",
}
`;
